pkg/checkmarx/reporting.go

```diff
@@ -51,22 +51,35 @@ type LowPerQuery struct {
 func CreateCustomReport(data map[string]interface{}, insecure, neutral []string) reporting.ScanReport {
 	deepLink := fmt.Sprintf(`<a href="%v" target="_blank">Link to scan in CX UI</a>`, data["DeepLink"])
 
+	var projectID, scanID, linesScanned, filesCcanned int64
+	if s, err := strconv.ParseInt(fmt.Sprint(data["ProjectId"]), 10, 64); err == nil {
+		projectID = s
+	}
+	if s, err := strconv.ParseInt(fmt.Sprint(data["ScanId"]), 10, 64); err == nil {
+		scanID = s
+	}
+	if s, err := strconv.ParseInt(fmt.Sprint(data["LinesOfCodeScanned"]), 10, 64); err == nil {
+		linesScanned = s
+	}
+	if s, err := strconv.ParseInt(fmt.Sprint(data["FilesScanned"]), 10, 64); err == nil {
+		filesCcanned = s
+	}
 	scanReport := reporting.ScanReport{
 		ReportTitle: "Checkmarx SAST Report",
 		Subheaders: []reporting.Subheader{
 			{Description: "Project name", Details: fmt.Sprint(data["ProjectName"])},
-			{Description: "Project ID", Details: fmt.Sprint(data["ProjectID"])},
+			{Description: "Project ID", Details: fmt.Sprint(projectID)},
```

Comment:
change - fmt.Sprint(data["ScanID"]) to fmt.Sprint(data["ScanId"])
change - fmt.Sprint(data["ScanID"]) to fmt.Sprint(data["ScanId"])
change - fmt.Sprint(data["LinesOfCodeScanned)"]) to fmt.Sprint(data["LinesOfCodeScanned"])
change - fmt.Sprint(data["FilesScanned)"]) to fmt.Sprint(data["FilesScanned"])